deploy/README.md

```diff
@@ -0,0 +1,309 @@
+```
+
+This way your changes are kept separate from the original WBS Deploying Kit code.
+
+### Managing your Data
+Besides [your configuration](#configuring-your-wikibase-suite), your data is obviously what makes your instance unique. All data is stored in [Docker Volumes](https://docs.docker.com/storage/volumes/).
+
+ - `wikibase-image-data` MediaWiki image and media file uploads.
+ - `mysql-data` MediaWiki/Wikibase MariaDB raw database.
+ - `wdqs-data` Wikidata Query Service raw database.
+ - `elasticsearch-data` Elastic Search raw database.
+ - `quickstatements-data` Generated Quickstatement OAuth binding for this MediaWiki instance.
+ - `traefik-letsencrypt-data` SSL certificates.
+
+#### Backup your Data
+You can backup your data by shutting down the instance and dumping the contents of all Docker Volumes into `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker run --rm --volume $v:/backup debian:12-slim tar cz backup > $v.tar.gz
+done
+```
+
+#### Restore from a backup
+
+To restore the volume backups, ensure your instance is shut down with `docker compose down` and populate the Docker Volumes with data from your `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker volume rm $v 2> /dev/null
+  docker volume create $v
+  docker run -i --rm --volume $v:/backup debian:12-slim tar xz < $v.tar.gz
+done
+```
+
+### Updating
+
+WBS is versioned with [Semantic Versioning](https://semver.org/spec/v2.0.0.html). The WBS Deployment Kit and all the WBS Service Images have individual version numbers.
+
+> 💡 WBS Deployment Kit always references the latest minor and patch releases of the compatible WBS Service Images major versions.
+
+#### Minor and patch updates for WBS Service Images
+
+As WBS Deployment Kit always references latest minor and patch releases of compatible WBS Service Images TODO: rephrase, non breaking changes (including security updates) are applied automatically when recreating Docker containers. This should be always safe to do. Simply run
+
+```sh
+docker compose down 
+docker compose up --wait
+```
+
+> 💡 If you do not want to pull in new versions of WBS Service Images on container restart, just stop your containers using `docker compose stop` before restart. This will keep the current containers intact. Please note that this will not apply any security updates. It is generally recommended to remove your containers for restart with `docker compose down`.
```

Comment:
Recommending using `down` as general practice is likely a good call, but I'm not certain about it. Do you feel sure about that?

Reply:
good question - how much docker knowledge are we assuming from the users here? If we assume they know the difference I would say: let's say sth like "use docker compose down if you want to remove your containers". Basically without recommending anything unless we think it is crucial for the good working of WBS. Does this resonate?

Reply:
Well, considering the containers as "a stateless instance of the code" (that is the docker image), it should be safe to always throw them away. Practicing that could also be a way to ensure that it is safe. 

We have config in `./config` now and all data in docker volumes. So I would vote for removing containers and use `down`. Otherwise I would frighten we could hide a dependency on state in the container we do not expect.

Reply:
In spirit I'm with Mónica on this one, but in practice I am ok with suggesting `down` as a general rule. I just think from a Docker perspective it is a little surprising, though not enough to warrant an explanation.

I feel there may be cases where re-provisioning the containers from scratch is ill advised, but other than it implying a longer restart cycle on the service, I can't think of any cases now.

However, to your point I do think we should be confident that our containers are stateless regardless, and that is something we should look to our test suite to adequately prove. Our images when used in another setup or orchestration layer (i.e. Kubernetes) may be stopped and started vs down'd as standard practice.

That's my thinking on this, but having said all that, I am ok with this as it is for now. Just things worth us keeping in mind.

Reply:
I mean, we do have the logs at the moment (`wikibase` container `/var/log/mediawiki`), which are written into the containers and would be lost on `down`.

Though, I'd rather put the logs in a volume or send them to stdout only than having the container carry state intentionally .

Reply:
So for this PR I think we can just go forward with `down`

Reply:
> I mean, we do have the logs at the moment (`wikibase` container `/var/log/mediawiki`), which are written into the containers and would be lost on `down`.
> 
> 
> 
> Though, I'd rather put the logs in a volume or send them to stdout only than having the container carry state intentionally .

Good point, and agreed. Clearing the nom-trivial accumulation of log files could actually be a reason for `down` on its own:)

Also agree that ideally the logs go to STDOUT if there is no volume, otherwise they go to files. 

However I did try getting them going to STDOUT and there was an issue with it due to how things work in MediaWiki (I think they essentially need to be bounced to a file with another process pushing them to STDOUT), and otherwise they are rather overwhelming to the otherwise (debatably) more fundamental Apache logs...

Note: Will copy a link you this to the related Phan ticket. Nothing to do here now 👍🏼